frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -536,177 +504,118 @@ export function Experiment(): JSX.Element {
+                        {showWarning &&
+                            experimentResults &&
+                            ((experimentInsightType === InsightType.TRENDS && areCountResultsSignificant) ||
+                                (experimentInsightType === InsightType.FUNNELS && areConversionResultsSignificant)) && (
+                                <Row align="middle" className="significant-results">
+                                    <Col span={19} style={{ color: '#497342' }}>
+                                        Experiment results are significant. You can end your experiment now or let it
+                                        run until completion.
+                                    </Col>
+                                    <Col span={5}>
+                                        <Button style={{ color: '#497342' }} onClick={() => setShowWarning(false)}>
+                                            Dismiss
+                                        </Button>
+                                    </Col>
+                                </Row>
                             )}
-                            <Col>
-                                <div className="card-secondary mt">Variants</div>
-                                <ul className="variants-list">
-                                    {experimentData.parameters?.feature_flag_variants?.map(
-                                        (variant: MultivariateFlagVariant, idx: number) => (
-                                            <li key={idx}>{variant.key}</li>
-                                        )
-                                    )}
-                                </ul>
-                            </Col>
-                            <Row>
-                                <Col className="mr">
-                                    <div className="card-secondary mt">Start date</div>
-                                    {experimentData.start_date ? (
-                                        <span>{dayjs(experimentData.start_date).format('D MMM YYYY')}</span>
-                                    ) : (
-                                        <span className="description">Not started yet</span>
-                                    )}
-                                </Col>
-                                {experimentData.end_date && (
-                                    <Col className="ml">
-                                        <div className="card-secondary mt">Completed date</div>
-                                        <span>{dayjs(experimentData.end_date).format('D MMM YYYY')}</span>
+                    </Row>
+                    <Row>
+                        <Collapse className="full-width" defaultActiveKey="experiment-details">
+                            <Collapse.Panel header={<b>Experiment details</b>} key="experiment-details">
+                                <ExperimentPreview
+                                    experiment={experimentData}
```

Comment:
these are on the result page, so shouldn't use the variable values 😅 , but rather `parameters.recommended*`